File: js/Account.js
class Account {
    
    constructor(name, document, email){
        this.id = id;
        this.name = name;
        this.document = document;
        this.email = email;
        this.password;
    }

    printDataCar() {
        console.log(this.name)
        console.log(this.document)
        console.log(this.email)
    }
}

export {Account}

File: python/Account.py
class Account:
    id: int
    name: str
    document: str
    email: str
    password: str

    def __init__(self, id: int, name: str, document: str, email: str, password: str) -> None:
        self.id = id
        self.name = name
        self.document = document
        self.email = email
        self.password = password